circuits/circom/verify_signature.circom

```diff
@@ -45,7 +45,7 @@ template EdDSAMiMCSpongeVerifier_patched() {
   hash.k <== 0;
```

Comment:
I still keep the MiMC sponge in the signature verification here. Let's change it when we have more knowledge on #98